src/ts/Document.ts

```diff
@@ -36,8 +36,6 @@ export interface BuildForeignKeyDefinitionsResult {
 class Document {
 	[key: string]: unknown;
 
-	public _id?: string;
```

Comment:
This is a little wonky and might need further refactor later.  A `Document` doesn't really have an `_id`, but a `Model` instance does.  However, there are multiple places in the `Document` class looking at the `_id`.  For now it'll just become part of the index signature and type checking will need to be more robust around it.